### PR TITLE
[sysvabi64] Define the processor specific interpretation for DT_PLTGOT

### PR DESCRIPTION
**Pull request by smithp35**
On AArch64 we define DT_PLTGOT to be the address of the .plt.got section.

### CODE REVIEW

File: sysvabi64/sysvabi64.rst [hunk 1]
```diff
@@ -984,9 +987,15 @@ static linker and the dynamic linker. There are a number of entries
...
 that are architecture independent.
 
+The generic dynamic tags are defined in SCO-ELF_ Dynamic Section.
+
 The AArch64 specific dynamic tags are defined in AAELF64_ Dynamic
 Section.
 
+The generic tag ``DT_PLTGOT`` tag has a processor specific
+implementation. On AArch64 it is defined to be the address of the
+``.plt.got`` section.
```
Comment (mmalcomson) [inline sysvabi64/sysvabi64.rst:997]: I think a typo here -- "generic tag `DT_PLTGOT` tag"  -- has duplication of the word `tag` .
Also typo `.plt.got -> .got.plt`.

Comment (smithp35) [reply sysvabi64/sysvabi64.rst:997]: Thank you! My apologies, if only my eyes saw what I wrote and not what I think I wrote.
Will fix.

**Comment by mmalcomson** [comment]
LGTM, do you mind squashing the commits into one?
I'll merge it then.

**Comment by smithp35** [comment]
Thank you, I've squashed the commits.

**Pull request merged**